docs/logging-format.md

```diff
@@ -228,6 +228,57 @@ and TLS probe points.
 | `tls::key_exchange_algorithm` | uint16     | Key exchange mode: ECDHE(0), DHE(1), PSK(2), ECDHE-PSK(3), DHE-PSK(4)                            |
 | `tls::group`                  | uint16     | Groups used in the handshake (as in IANA [registry][iana-tls-supported-groups])                  |
 
+##### SSH context names
+
+| name                   | description                            |
+|------------------------|----------------------------------------|
+| `ssh::handshake_client`| SSH handshake for client               |
+| `ssh::handshake_server`| SSH handshake for server               |
+| `ssh::client_key`      | SSH client key signature/verification  |
+| `ssh::server_key`      | SSH server key signature/verification  |
+| `ssh::key_exchange`    | SSH key exchange                       |
+
+##### SSH keys
+
+All the keys except `rsa_bits` have `string` type.
+We distinguish server and client values by the context we are in. We log all relevant events in both contexts.
+
+| key                             | description                                      | example                    |
+|---------------------------------|--------------------------------------------------|----------------------------|
+| `ssh::ident_string`             | Software identification string                   | `SSH-2.0-OpenSSH_8.8`      |
+| `ssh::peer_ident_string`        | Peer software identification string              | `SSH-2.0-OpenSSH_8.8`      |
+| `ssh::key_algorithm`            | Key used in handshake/key ownership proof        | `ssh-ed25519`              |
+| `ssh::rsa_bits`                 | Key bits (RSA only)                              | 2048                       |
+| `ssh::cert_signature_algorithm` | If cert is used, signature algorithm of the cert | `ecdsa-sha2-nistp521`      |
+| `ssh::kex_algorithm`            | Negotiated key exchange algorithm                | `curve25519-sha256`        |
+| `ssh::kex_group`                | Group used for key exchange                      | moduli+bits or group name. |
+| `ssh::c2s_cipher`               | Data cipher algorithm                            | `aes256-gcm@openssh.com`   |
+| `ssh::s2c_cipher`               |                                                  |                            |
+| `ssh::c2s_mac`                  | Data integrity algorithm                         | Omitted for "implicit"     |
+| `ssh::s2c_mac`                  |                                                  |                            |
+| `ssh::c2s_compression`          | Data compression algorithm                       | Omitted for "none"         |
+| `ssh::s2c_compression`          |                                                  |                            |
```

Comment:
maybe one more nit here that you have `Omitted for ..` in the `example` column, while it would be more suitable in the `description` column and keeping the `implicit`/`none` in the example column only.

Reply:
Thanks, done